docs/en_US/NAS/SPOS.rst

```diff
@@ -61,15 +57,13 @@ NOTE: The data loading used in the official repo is `slightly different from usu
 Step 2. Evolution Search
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Single Path One-Shot leverages evolution algorithm to search for the best architecture. The tester, which is responsible for testing the sampled architecture, recalculates all the batch norm for a subset of training images, and evaluates the architecture on the full validation set.
-
-In order to make the tuner aware of the flops limit and have the ability to calculate the flops, we created a new tuner called ``EvolutionWithFlops`` in ``tuner.py``\ , inheriting the tuner in SDK.
+Single Path One-Shot leverages evolution algorithm to search for the best architecture. The search module, which is responsible for testing the sampled architecture, recalculates all the batch norm for a subset of training images, and evaluates the architecture on the full validation set.
```

Comment:
We should note here that this is what has been done in paper, not what we have implemented.